tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/ProcessZeebeRecordProcessorElasticSearch.java

```diff
@@ -127,6 +127,7 @@ private ProcessEntity createEntity(
     processEntity.setBpmnProcessId(process.getBpmnProcessId());
     processEntity.setVersion(process.getVersion());
     processEntity.setTenantId(process.getTenantId());
+    processEntity.setBpmnXml(new String(process.getResource()));
```

Comment:
[q] Should we set the character encoding on the String declaration over here?
`processEntity.setBpmnXml(new String(process.getResource(), StandardCharsets.UTF_8));`
 to avoid potential issues with character encoding

Reply:
I don't think we need to do explicitely, it's already defined UTF as default.